requirements/test.txt

```diff
@@ -4,57 +4,53 @@
 #
 #    make upgrade
 #
-atomicwrites==1.3.0       # via pytest
 attrs==19.3.0             # via pytest
 certifi==2019.11.28
 chardet==3.0.4
-click==7.0                # via code-annotations
+click==7.1.1              # via code-annotations
 code-annotations==0.3.3
-configparser==4.0.2       # via importlib-metadata
-contextlib2==0.6.0.post1  # via importlib-metadata
-coverage==5.0             # via pytest-cov
+coverage==5.0.3           # via pytest-cov
 django-crum==0.7.5
 django-model-utils==3.2.0
-django-waffle==0.18.0
-djangorestframework-jwt==1.11.0
-djangorestframework==3.9.4
-edx-django-utils==2.0.2
-edx-drf-extensions==2.4.5
+django-waffle==0.20.0
+djangorestframework==3.11.0
+drf-jwt==1.15.1
```

Comment:
There is a breaking change in `version>1.15.0`. See https://github.com/edx/portal-designer/pull/84 for reference. You'll need to add some installed applications and set the default to `JWT`.

Reply:
done